Fix commit hash discovery (#107)
sonatype-nexus-community/scan-gradle-plugin: README.md

    modulesExcluded = ['module-1', 'module-2'] // Optional. For multi-module projects, the names of the sub-modules to exclude from scanning and evaluation.
    dirExcludes = 'some-ant-pattern' // Optional. Comma separated ant-like glob patterns to select directories/archives that should be excluded. For Android projects we suggest using '**/classes.jar,**/annotations.zip,**/lint.jar,**/internal_impl-*.jar'
    dirIncludes = 'some-ant-pattern' // Optional. Comma separated ant-like glob patterns to select directories/archives that should be examined

>     scanFolderPath = '/path/to/project/root' // Optional. Path to the project root, under which the '.git' folder usually resides. Defaults to the working directory (where gradle is executed).  

Reviewer: ```suggestion
    scanFolderPath = '/path/to/project/root' // Optional. Path to the project root, under which the '.git' folder usually resides. Defaults to the project's root directory.
```

Author: Same as above

    modulesExcluded = listOf("module-1", "module-2") // Optional. For multi-module projects, the names of the sub-modules to exclude from scanning and evaluation.
    dirExcludes = "some-ant-pattern" // Optional. Comma separated ant-like glob patterns to select directories/archives that should be excluded. For Android projects we suggest using "**/classes.jar,**/annotations.zip,**/lint.jar,**/internal_impl-*.jar"
    dirIncludes = "some-ant-pattern" // Optional. Comma separated ant-like glob patterns to select directories/archives that should be examined

>     scanFolderPath = "/path/to/project/root" // Optional. Path to the project root, under which the '.git' folder usually resides. Defaults to the working directory (where gradle is executed).

Reviewer: ```suggestion
    scanFolderPath = "/path/to/project/root" // Optional. Path to the project root, under which the '.git' folder usually resides. Defaults to the project's root directory.
```

Author: This does not reflect my suggestion (see comment above).

Author: I don't think we need to document it at all. The project's root directory is what we need. Users can only mess it up.

Reviewer: Agreed!
